datalad_next/iter_collections/tests/test_iterzip.py

```diff
@@ -87,4 +88,16 @@ def test_iter_zip(sample_zip):
         # do not compare mtime
         r.mtime = None
     for t in targets:
+        print(t.name)
```

Comment:
I suspect that this could/should be removed again.

Reply:
That is correct. Thanks. Done in 9b44e57a6fab29af7b1c864ac8ad05f6637d161c